src/main/java/io/github/bedwarsrel/BedwarsRel/Shop/Specials/ProtectionWallListener.java

```diff
@@ -79,6 +79,7 @@ public void onPlace(BlockPlaceEvent place) {
 
     place.setBuild(false);
     place.setCancelled(true);
+    return;

```

Comment:
Der Block bleibt nur da, wenn man ihn platziert. Wenn man die Mauer in der Luft "setzt" ist auch kein Block da. Aber das tatsächliche setzen müsste gefixt werden...